.github/workflows/shark_model_suite.yml

```diff
@@ -0,0 +1,91 @@
+# Runs the shark model suite on iree wheels from build package workflow
+name: Validate Shark Model Suite
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_id:
+        description: 'Release id to publish'
+        required: true
+      package_version:
+        description: 'Version of the package'
+        required: true
+      build_run_id:
+        description: 'Run ID for the build_package.yml workflow that triggered this workflow'
+        required: true
+
+jobs:
+  build-validate:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [a100, cpu, gpu, Linux, postsubmit, self-hosted, trust=basic, X64]
```

Comment:
These aren't OS's. The tags for GitHub runners allow you to target the intersection of tags. You should probably make this a "runner" or "runs-on" parameter and then each entry needs to be either a string or a list of strings. i.e. the renaming I did in #9432 is still good.